lib/camerawesome_plugin.dart

```diff
@@ -278,11 +260,7 @@ class CamerawesomePlugin {
     try {
```

Comment:
Do we still need this method here?

Reply:
Permission check is still in there, requesting permissions is removed.